libs/langchain/langchain/vectorstores/opensearch_vector_search.py

```diff
@@ -555,7 +555,7 @@ def similarity_search_with_score(
         return documents_with_scores
 
     def _raw_similarity_search_with_score(
```

Comment:
since this is only used in two places, would it make sense to update the other place to also pass in embeddings? that way we could make the method signature require fewer args and be tighter